axis.go

```diff
@@ -361,7 +361,12 @@ func (DefaultTicks) Ticks(min, max float64) (ticks []Tick) {
 	val := math.Floor(min/majorDelta) * majorDelta
 	for val <= max {
 		if val >= min && val <= max {
-			ticks = append(ticks, Tick{Value: val, Label: fmt.Sprintf("%g", float32(val))})
+			round := float32(val)
+			if val < 1 && val > -1 {
+				round = float32(val + 1)
+				round -= 1
+			}
+			ticks = append(ticks, Tick{Value: val, Label: fmt.Sprintf("%g", round)})
```

Comment:
Well, first, this is trying to be a minimal change to the existing code. What values the ticks are actually at is a separate matter.

Second, probably not. if you float64(round) you get values that are less close to the tick than the original value.